packages/fluent/scss/core/helpers/_layout.scss

```diff
@@ -16,6 +16,11 @@
     flex: 1 0 0%;
 }
 
+.k-flex-row { flex-direction: row; }
+.k-flex-row-reverse { flex-direction: row-reverse; }
+.k-flex-col { flex-direction: column; }
+.k-flex-col-reverse { flex-direction: column-reverse; }
+
```

Comment:
This should not be added - those helper utilities are [already available in the utils package](https://github.com/telerik/kendo-themes/blob/develop/packages/fluent/scss/utils/_layout.scss), which is used in the fluent theme.